metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -329,12 +330,17 @@ def get_log_stream(job):
                         else:
                             pass
                     else:
+                        i = 0
                         yield line
+                return
             except Exception as ex:
+                exception = ex
                 if self.is_crashed:
                     break
-                sys.stderr.write(repr(ex))
-                time.sleep(2 ** i)
+                #sys.stderr.write(repr(ex) + '\n')
```

Comment:
Why is this commented out?

Reply:
https://github.com/Netflix/metaflow/pull/197#discussion_r417683154